lib/custom/src/MShop/Service/Provider/Payment/OmniPay.php

```diff
@@ -545,6 +546,13 @@ public function updatePush( \Psr\Http\Message\ServerRequestInterface $request, \
 	 */
 	public function updateSync( \Psr\Http\Message\ServerRequestInterface $request, \Aimeos\MShop\Order\Item\Iface $order )
 	{
+		$base = $this->getOrderBase( $order->getBaseId() );
```

Comment:
These lines seems to be duplicate

Reply:
Hmm... Didn't touch the updateSync() method. Don't see duplicate lines. What should I do now?

Reply:
Can you make sure the implementation in your fork and PR is the same as in the current master branch so the unit tests doesn't fail any more:
https://github.com/aimeoscom/ai-payments/pull/12#discussion_r181494156

Reply:
now we have deleted all new code from Omnipay.php, so it will stay original like at master branch I think